os2borgerpc_client/bin/register_new_os2borgerpc_client.sh

```diff
@@ -112,47 +129,53 @@
         read DISTRO
     fi
 
-    if [[ -z $DISTRO ]]
+    if [[ -z "$DISTRO" ]]
     then
         echo "Indtast ID for PC'ens distribution"
         read DISTRO
     fi
 
-    echo "Distributions ID: "$DISTRO
+    echo "Distributions ID: $DISTRO"
 
-    sudo set_bibos_config distribution $DISTRO
+    set_os2borgerpc_config distribution "$DISTRO"
 
 
     # - mac
     #   Get the mac-address
-    sudo set_bibos_config mac `ip addr | grep link/ether | awk 'FNR==1{print $2}'`
+    set_os2borgerpc_config mac `ip addr | grep link/ether | awk 'FNR==1{print $2}'`
 
 
     # - admin_url
     #   Get from gateway, otherwise prompt user.
+    unset ADMIN_URL
     if [[ -n "$HAS_GATEWAY" ]]
     then
-        ADMIN_URL=$(get_bibos_config admin_url "$SHARED_CONFIG")
+        ADMIN_URL=$(get_os2borgerpc_config admin_url "$SHARED_CONFIG")
     fi
     if [[ -z "$ADMIN_URL" ]]
-    then 
+    then
         ADMIN_URL="https://bibos-admin.magenta-aps.dk"
         echo "Indtast admin-url hvis det ikke er $ADMIN_URL"
         read NEW_ADMIN_URL
-        if [[ -n $NEW_ADMIN_URL ]]
+        if [[ -n "$NEW_ADMIN_URL" ]]
         then
-            ADMIN_URL=$NEW_ADMIN_URL
+            ADMIN_URL="$NEW_ADMIN_URL"
         fi
     fi
-    sudo set_bibos_config admin_url $ADMIN_URL
+    set_os2borgerpc_config admin_url "$ADMIN_URL"
 
-    # OK, we got the config. 
+    # OK, we got the config.
     # Do the deed.
-    sudo bibos_register_in_admin
+    if ! os2borgerpc_register_in_admin; then
+        fatal "Tilmelding mislykkedes" && continue || exit 1
+    fi
 
     # Now setup cron job
     if [[ -f $(which jobmanager) ]]
     then
-        sudo echo 'PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin' > /etc/cron.d/bibos-jobmanager
-        sudo echo "*/5 * * * * root $(which jobmanager)" >> /etc/cron.d/bibos-jobmanager
+        echo 'PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin' > /etc/cron.d/os2borgerpc-jobmanager
+        echo "*/5 * * * * root $(which jobmanager)" >> /etc/cron.d/os2borgerpc-jobmanager
```

Comment:
We have a script that randomizes the 5 minute interval: https://bibos-admin.magenta-aps.dk/site/hvidovre/scripts/355/

Could we reuse it here?

Reply:
I suppose we could, but I think I'm fine keeping the existing behaviour for now...

@agger-magenta, what do you think?

Reply:
I agree with Alec.